Sources/FuzzilliCli/main.swift

```diff
@@ -217,6 +220,14 @@ if exportStatistics && storagePath == nil {
     configError("--exportStatistics requires --storagePath")
 }
 
+if args.double(for: "--statisticsExportInterval") == nil || statisticsExportInterval <= 0.0 {
```

Comment:
I think you can just drop the `args.double(for: "--statisticsExportInterval") == nil ||` part entirely since the default value is valid